website/docs/configuration/securing-access-to-the-dashboard.mdx

```diff
@@ -128,15 +122,15 @@ b'$2b$12$nLfl7lKBiYzgAN2aI3ii6exZSZ9KRsj18C7CEWY8kscj9.c6bRXim'
 Now create a Kubernetes secret to store your chosen username and the password hash:
 
 ```sh
-kubectl create secret generic admin-password-hash \
-  --namespace wego-system \
-  --from-literal=username=admin
+kubectl create secret generic cluster-user-auth \
+  --namespace flux-system \
+  --from-literal=username=admin \
   --from-literal=password='$2b$12$nLfl7lKBiYzgAN2aI3ii6exZSZ9KRsj18C7CEWY8kscj9.c6bRXim'
 ```
 
 You should now be able to login via the cluster user account using your chosen username and password. Follow the instructions in the next section in order to configure RBAC correctly.
 
-### RBAC authorization
+## RBAC authorization
```

Comment:
This whole section needs updating.

I'd replace the whole thing by only the role we created in the quickstart in point 5 of https://staging.docs.gitops.weave.works/v2-restructure/docs/next/getting-started#configure-access-to-the-dashboard